server/commands/hubs.py

```diff
@@ -575,11 +576,41 @@ def ooc_cmd_unhide_clients(client, arg):
     )
 
 
+@mod_only(hub_owners=True)
+def ooc_cmd_follow_me(client, arg):
+    """
+    Force someone to follow you. Follow me!
+    Usage: /follow_me <id>
+    """
+    if len(arg) == 0:
+        raise ArgumentError("You must specify a target. Use /follow_me <id>.")
+    try:
+        targets = client.server.client_manager.get_targets(
+            client, TargetType.ID, int(arg), False
+        )
+    except:
+        raise ArgumentError("You must specify a target. Use /follow_me <id>.")
+    if targets:
+        for c in targets:
+            if client == c:
+                raise ClientError("You are already forced to follow yourself because you are yourself!")
+            c.following = client
+            c.forced_to_follow = True
+            c.send_ooc(f"You've been forced to follow {client.showname}!")
+            if c.area != client.area:
+                c.set_area(client.area)
+        client.send_ooc(f"Forced {len(targets)} existing client(s) to follow you.")
+    else:
+        client.send_ooc("No targets found.")
+
+
 def ooc_cmd_follow(client, arg):
     """
     Follow targeted character ID.
     Usage: /follow [id]
     """
+    if client.forced_to_follow and not client.is_mod and client not in client.area.area_manager.owners:
```

Comment:
This should check if .following still exists, because the client you're following may disconnect, get deleted, etc.